Automattic/wp-calypso
# Add handling for currency URL parameter to manually change currencies

## Proposed Changes

Check for currency URL parameter and use that as priority over user currency and product currency
Check for this in product list, lightbox price, and plan price breakdown

## Testing Instructions

1. Checkout this branch or use the Calypso Green live link
2. Checkout this wpcom diff: D131767-code
3. Sandbox `public-api.wordpress.com`
4. Go to `/pricing`
5. Make sure the currency is what you normally have it set to
![image](https://github.com/Automattic/wp-calypso/assets/65001528/1fa585f5-b77f-4187-af23-3e46393f4334)
6. Now go to `/pricing?currency=AUD` and make sure the pricing updates to AUD prices
![image](https://github.com/Automattic/wp-calypso/assets/65001528/4bd9719e-40c1-4410-867b-699cd1bd25f5)
7. Try this with several other currencies and make sure all the prices are correct. Make sure to check the product lightboxes as well as the pricing breakdown in the lightboxes for the plans
![image](https://github.com/Automattic/wp-calypso/assets/65001528/0774cdd4-c6e1-4096-a3d2-b5aa2cc4c794)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/e6100d4b-a58f-4830-bb95-4f7fb8ec3538)

## Pre-merge Checklist
